### DISCUSSION TITLE
Bunch of channel to be sorted out

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#194] [Show and tell]
some are albanian or turkey  some italian  channel   i cant tell 

#EXTINF:0,xDesire XXX
http://185.226.88.3:1036/live/backup/backup/7.m3u8
#EXTINF:0,xBungd XXX
http://185.226.88.3:1036/live/backup/backup/8.m3u8
#EXTINF:0,xBizare XXX
http://185.226.88.3:1036/live/backup/backup/9.m3u8

#EXTINF:0,Fen Tv 
http://185.226.88.3:1036/live/backup/backup/12.m3u8
#EXTINF:0,Balkanika 
http://185.226.88.3:1036/live/backup/backup/14.m3u8
#EXTINF:0,EuroStar 
http://185.226.88.3:1036/live/backup/backup/22.m3u8
#EXTINF:0,ZDF 
http://185.226.88.3:1036/live/backup/backup/38.m3u8
#EXTINF:0,Deutsche Welle 
http://185.226.88.3:1036/live/backup/backup/39.m3u8
#EXTINF:0,RTL 2 
http://185.226.88.3:1036/live/backup/backup/41.m3u8
#EXTINF:0,RTL 
http://185.226.88.3:1036/live/backup/backup/42.m3u8
#EXTINF:0,RaiNews 24 
http://185.226.88.3:1036/live/backup/backup/44.m3u8
#EXTINF:0,Rai Scuola 
http://185.226.88.3:1036/live/backup/backup/46.m3u8
#EXTINF:0,Rai Gulp 
http://185.226.88.3:1036/live/backup/backup/47.m3u8
#EXTINF:0,Rai Tre 
http://185.226.88.3:1036/live/backup/backup/50.m3u8
#EXTINF:0,Mediaset Italia 
http://185.226.88.3:1036/live/backup/backup/51.m3u8
#EXTINF:0,Report TV 
http://185.226.88.3:1036/live/backup/backup/52.m3u8
#EXTINF:0,Living HD 
http://185.226.88.3:1036/live/backup/backup/56.m3u8
#EXTINF:0,Shenja Tv 
http://185.226.88.3:1036/live/backup/backup/57.m3u8
#EXTINF:0,Tring Sport 3 HD 
http://185.226.88.3:1036/live/backup/backup/61.m3u8
#EXTINF:0,Tring Sport 2 HD 
http://185.226.88.3:1036/live/backup/backup/62.m3u8
#EXTINF:0,Tring Sport 1 HD 
http://185.226.88.3:1036/live/backup/backup/63.m3u8
#EXTINF:0,Tring International 
http://185.226.88.3:1036/live/backup/backup/64.m3u8
#EXTINF:0,Tring Sport News 
http://185.226.88.3:1036/live/backup/backup/65.m3u8
#EXTINF:0,Tring World 
http://185.226.88.3:1036/live/backup/backup/66.m3u8
#EXTINF:0,Tring Planet 
http://185.226.88.3:1036/live/backup/backup/67.m3u8
#EXTINF:0,Tring History 
http://185.226.88.3:1036/live/backup/backup/68.m3u8
#EXTINF:0,Tring Action 
http://185.226.88.3:1036/live/backup/backup/69.m3u8
#EXTINF:0,Tring Comedy 
http://185.226.88.3:1036/live/backup/backup/70.m3u8
#EXTINF:0,Tring Super 
http://185.226.88.3:1036/live/backup/backup/71.m3u8
#EXTINF:0,Tring Fantasy 
http://185.226.88.3:1036/live/backup/backup/72.m3u8
#EXTINF:0,Tring Life 
http://185.226.88.3:1036/live/backup/backup/73.m3u8
#EXTINF:0,Tring Shqip 
http://185.226.88.3:1036/live/backup/backup/74.m3u8
#EXTINF:0,Jolly HD 
http://185.226.88.3:1036/live/backup/backup/75.m3u8
#EXTINF:0,3 Plus 
http://185.226.88.3:1036/live/backup/backup/76.m3u8
#EXTINF:0,Tip TV 
http://185.226.88.3:1036/live/backup/backup/77.m3u8
#EXTINF:0,Tring Tring 
http://185.226.88.3:1036/live/backup/backup/78.m3u8
#EXTINF:0,Tring Kids 
http://185.226.88.3:1036/live/backup/backup/79.m3u8
#EXTINF:0,Muse
http://185.226.88.3:1036/live/backup/backup/80.m3u8
#EXTINF:0,Kanali 7 
http://185.226.88.3:1036/live/backup/backup/81.m3u8
#EXTINF:0,FaX NEWS 
http://185.226.88.3:1036/live/backup/backup/82.m3u8
#EXTINF:0,Zjarr TV 
http://185.226.88.3:1036/live/backup/backup/83.m3u8
#EXTINF:0,Apollon TV 
http://185.226.88.3:1036/live/backup/backup/84.m3u8
#EXTINF:0,Tirana1 
http://185.226.88.3:1036/live/backup/backup/85.m3u8
#EXTINF:0,Syri 
http://185.226.88.3:1036/live/backup/backup/86.m3u8
#EXTINF:0,Folk Plus 
http://185.226.88.3:1036/live/backup/backup/87.m3u8
#EXTINF:0,Elrodi 
http://185.226.88.3:1036/live/backup/backup/88.m3u8
#EXTINF:0,Koplik TV 
http://185.226.88.3:1036/live/backup/backup/89.m3u8
#EXTINF:0,Click TV 
http://185.226.88.3:1036/live/backup/backup/90.m3u8
#EXTINF:0,Club TV 
http://185.226.88.3:1036/live/backup/backup/92.m3u8
#EXTINF:0,Tema TV 
http://185.226.88.3:1036/live/backup/backup/93.m3u8
#EXTINF:0,Kukesi TV 
http://185.226.88.3:1036/live/backup/backup/94.m3u8
#EXTINF:0,Adrianet 
http://185.226.88.3:1036/live/backup/backup/95.m3u8
#EXTINF:0,Smile 
http://185.226.88.3:1036/live/backup/backup/96.m3u8
#EXTINF:0,Italia Uno 
http://185.226.88.3:1036/live/backup/backup/102.m3u8
#EXTINF:0,Canale 5 
http://185.226.88.3:1036/live/backup/backup/103.m3u8
#EXTINF:0,Rai Uno 
http://185.226.88.3:1036/live/backup/backup/105.m3u8
#EXTINF:0,Klan HD 
http://185.226.88.3:1036/live/backup/backup/106.m3u8
#EXTINF:0,Klan Plus
http://185.226.88.3:1036/live/backup/backup/107.m3u8
#EXTINF:0,Klan Macedonia 
http://185.226.88.3:1036/live/backup/backup/109.m3u8
#EXTINF:0,Klan Kosova 
http://185.226.88.3:1036/live/backup/backup/110.m3u8
#EXTINF:0,Vizion Plus HD 
http://185.226.88.3:1036/live/backup/backup/111.m3u8
#EXTINF:0,BBF 
http://185.226.88.3:1036/live/backup/backup/112.m3u8
#EXTINF:0,ORA News 
http://185.226.88.3:1036/live/backup/backup/113.m3u8
#EXTINF:0,Alsat-M 
http://185.226.88.3:1036/live/backup/backup/116.m3u8
#EXTINF:0,RTSH 24 
http://185.226.88.3:1036/live/backup/backup/117.m3u8
#EXTINF:0,RTSH1 HD 
http://185.226.88.3:1036/live/backup/backup/118.m3u8
#EXTINF:0,RTSH2 HD 
http://185.226.88.3:1036/live/backup/backup/119.m3u8
#EXTINF:0,RTSH3 HD 
http://185.226.88.3:1036/live/backup/backup/120.m3u8
#EXTINF:0,RTSH Plus 
http://185.226.88.3:1036/live/backup/backup/121.m3u8
#EXTINF:0,RTSH Film 
http://185.226.88.3:1036/live/backup/backup/122.m3u8
#EXTINF:0,RTSH Shqip 
http://185.226.88.3:1036/live/backup/backup/123.m3u8
#EXTINF:0,RTSH Sport 
http://185.226.88.3:1036/live/backup/backup/124.m3u8
#EXTINF:0,RTSH Korca 
http://185.226.88.3:1036/live/backup/backup/125.m3u8
#EXTINF:0,RTSH Gjirokastra 
http://185.226.88.3:1036/live/backup/backup/126.m3u8
#EXTINF:0,RTSH Kuvend 
http://185.226.88.3:1036/live/backup/backup/127.m3u8
#EXTINF:0,RTSH Muzike 
http://185.226.88.3:1036/live/backup/backup/128.m3u8
#EXTINF:0,RTSH Femije 
http://185.226.88.3:1036/live/backup/backup/129.m3u8
#EXTINF:0,Top Channel HD 
http://185.226.88.3:1036/live/backup/backup/131.m3u8
#EXTINF:0,BBC 
http://185.226.88.3:1036/live/backup/backup/132.m3u8
#EXTINF:0,Al Jazeera English 
http://185.226.88.3:1036/live/backup/backup/133.m3u8
#EXTINF:0,CNN 
http://185.226.88.3:1036/live/backup/backup/134.m3u8
#EXTINF:0,RTK 
http://185.226.88.3:1036/live/backup/backup/135.m3u8
#EXTINF:0,KTV 
http://185.226.88.3:1036/live/backup/backup/136.m3u8
#EXTINF:0,Alj Balkan 
http://185.226.88.3:1036/live/backup/backup/138.m3u8
#EXTINF:0,Ora HD 
http://185.226.88.3:1036/live/backup/backup/139.m3u8
#EXTINF:0,Channel One 
http://185.226.88.3:1036/live/backup/backup/140.m3u8
#EXTINF:0,A2 News 
http://185.226.88.3:1036/live/backup/backup/141.m3u8
#EXTINF:0,Russia Today 
http://185.226.88.3:1036/live/backup/backup/142.m3u8
#EXTINF:0,ANT1 
http://185.226.88.3:1036/live/backup/backup/144.m3u8
#EXTINF:0,TGCOM24 
http://185.226.88.3:1036/live/backup/backup/146.m3u8
#EXTINF:0,La 7 
http://185.226.88.3:1036/live/backup/backup/147.m3u8
#EXTINF:0,Cartoon Netw 
http://185.226.88.3:1036/live/backup/backup/149.m3u8
#EXTINF:0,Fashion TV 
http://185.226.88.3:1036/live/backup/backup/150.m3u8
#EXTINF:0,Boomerang 
http://185.226.88.3:1036/live/backup/backup/151.m3u8
#EXTINF:0,Real Time 
http://185.226.88.3:1036/live/backup/backup/154.m3u8
#EXTINF:0,ATV Avrupa 
http://185.226.88.3:1036/live/backup/backup/155.m3u8
#EXTINF:0,D Drama 
http://185.226.88.3:1036/live/backup/backup/157.m3u8
#EXTINF:0,News 24 
http://185.226.88.3:1036/live/backup/backup/159.m3u8
#EXTINF:0,Abc News 
http://185.226.88.3:1036/live/backup/backup/160.m3u8
#EXTINF:0,TV Johaniter 
http://185.226.88.3:1036/live/backup/backup/162.m3u8
#EXTINF:0,RTSH Shkolla 
http://185.226.88.3:1036/live/backup/backup/161.m3u8
#EXTINF:0,Drita TV 
http://185.226.88.3:1036/live/backup/backup/163.m3u8
#EXTINF:0,EuroNews 
http://185.226.88.3:1036/live/backup/backup/164.m3u8
#EXTINF:0,Top Channel pa traskod 
http://185.226.88.3:1036/live/backup/backup/165.m3u8
#EXTINF:0,KLAN PA 
http://185.226.88.3:1036/live/backup/backup/166.m3u8
#EXTINF:0,Klan News
http://185.226.88.3:1036/live/backup/backup/167.m3u8
#EXTINF:0,NTV
http://185.226.88.3:1036/live/backup/backup/168.m3u8
#EXTINF:0,1 HD
http://185.226.88.3:1036/live/backup/backup/178.m3u8

tring tv list

#EXTINF:-1,ATV 
http://tring.mediazonehd.com:8880/ATV/video.m3u8
#EXTINF:-1,RTSH Shkollë 
http://tring.mediazonehd.com:8880/rtshshkollesRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Alsat M 
http://tring.mediazonehd.com:8880/alsatmsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Kanal 10 
http://tring.mediazonehd.com:8880/Kanal10sRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,ABC News 
http://tring.mediazonehd.com:8880/abcnewssRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Kanali 7 
http://tring.mediazonehd.com:8880/kanal7uaishiasfilahsf7887888/video.m3u8
#EXTINF:-1,RTSH 24 
http://tring.mediazonehd.com:8880/rtsh24sRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,RTSH 3 
http://tring.mediazonehd.com:8880/rtsh3sRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Living 
http://tring.mediazonehd.com:8880/livinghdsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,KanalD Drama 
http://tring.mediazonehd.com:8880/kanalddramasRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Jolly HD 
http://tring.mediazonehd.com:8880/jollyhdsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,3 Plus 
http://tring.mediazonehd.com:8880/3plussRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Smile 
http://tring.mediazonehd.com:8880/tringsmilesRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Tring Shqip 
http://tring.mediazonehd.com:8880/tringshqipsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Tring Super 
http://tring.mediazonehd.com:8880/tringsupersRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Tring World 
http://tring.mediazonehd.com:8880/tringworldsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Klan TV 
http://tring.mediazonehd.com:8880/klansRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,RTK 1 
http://tring.mediazonehd.com:8880/rtk1hdsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Vizion Plus 
http://tring.mediazonehd.com:8880/vizionplussRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Tring Tring 
http://tring.mediazonehd.com:8880/tringtringsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Tip TV 
http://tring.mediazonehd.com:8880/tiptvsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Tring Comedy 
http://tring.mediazonehd.com:8880/tringcomedysRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Tring Planet 
http://tring.mediazonehd.com:8880/tringplanetsRQbXDJb9wacRvXLe2cz/video.m3u8
#EXTINF:-1,Report TV
http://tring.mediazonehd.com:8880/ReportTVsRQbXDJb9wacRvXLe2cz/video.m3u8

**Comment by Agumonz98** [comment]
Tring Tring doesn't seem to work anymore (link: http://tring.mediazonehd.com:8880/tringtringsRQbXDJb9wacRvXLe2cz/video.m3u8) and the backup stream is pretty unstable.

**Comment by Sphinxroot** [comment]
yes  seem that all http://tring.mediazonehd.com:8880  are gone  and http://185.226.88.3:1036/ play well for me

**Reply by Agumonz98** [reply]
For me the http://185.226.88.3:1036/ stream constantly buffers, freezes or resets. So I hope someone finds a stable stream like the http://tring.mediazonehd.com:8880 one.

**Comment by BellezaEmporium** [comment]
Xtream-Codes server sadly, even if for now it still works perfectly.

https://ghostbin.com/q8jQK/Lane <- Panel dump
